microcache/__init__.py

```diff
@@ -304,6 +307,10 @@ def enable():
     return CACHE_OBJ.enable()
 
 
+def items():
+    return CACHE_OBJ.items()
+
```

Comment:
Would it be worth being able to filter down to a subset of the cache? Seems like it could be useful when the cache is full of hierarchical data / data with predictable path keys.

Reply:
Sounds like a reasonable feature to me. Implemented that and also filtered out expired items.

Reply:
I like it!